### ISSUE TITLE
data files are out of order

### ISSUE THREAD

**Issue by ChillZoe** [bobwan1995/PMFNet#26]
The image ids in file "hoi_data/coco/vcoco/annotations/addOnlySeg_instances_with_keypoints_vcoco_test_2014.json_with _keypoints" belongs to val data of vcoco, and the image ids in file "hoi_data/coco/vcoco/annotations/addOnlySeg_instances_with_keypoints_vcoco_val_2014.json_with _keypoints"  belongs to training data of vcoco , so where is the test data and how to reproduce the results on vcoco test set?
Thanks

**Comment by bobwan1995** [comment]
We pick out the predicted keypoints w.r.t. to image ids. All the key-points are predicted with pre-trained CPN on coco 2014 trainset.
